common/cards/alter-egos-iii/hermits/poepoeskizz-rare.ts

```diff
@@ -37,7 +36,7 @@ class PoePoeSkizzRare extends Card {
 	override onAttach(game: GameModel, component: CardComponent, observer: ObserverComponent) {
 		const {player} = component
 
-		observer.subscribe(player.hooks.onAttack, (attack) => {
+		observer.subscribe(player.hooks.beforeAttack, (attack) => {
```

Comment:
This should probably be in `afterAttack` to check if the attacker is still alive before picking to move

Reply:
Thats true. I think this should adjusted to do that.